alembic/versions/24533471e302_add_the_verbose_column.py

```diff
@@ -0,0 +1,23 @@
+"""Add the verbose column.
+
+Revision ID: 24533471e302
+Revises: 59ee93c4bf71
+Create Date: 2015-03-20 14:49:57.753166
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '24533471e302'
+down_revision = '59ee93c4bf71'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('preferences',
+                  sa.Column('verbose', sa.Boolean(), server_default="TRUE"))
```

Comment:
> whose value for this column

But, there can't be a value for this column, because it is being add on this same line.

(I think) there is a difference between ``default=...`` and ``server_default=...``.

- With ``default``, you create the column, and then tell ``sqlalchemy`` to set default values afterwards (but they will originally have null values on the DB side).
- With ``server_default``, you instruct the DB (postgres) to create the column with this default in the table definition.  It shouldn't be possible to have null values in the first place.



> I have been bitten by this before

In those cases, did you use ``default`` or ``server_default``?